contrib/emacs/hurl-mode.el

```diff
@@ -0,0 +1,136 @@
+;;; hurl-mode.el --- Major mode for hurl  -*- lexical-binding: t; -*-
+
+;; Copyright (C) 2022 Azzam S.A
+
+;; Author: Azzam S.A <vcs@azzamsa.com>
+;; Homepage: https://github.com/Orange-OpenSource/hurl
+;; Keywords: Hurl, shell
+
+;; Package-Version: 0.1.0
+;; Package-Requires: ((emacs "24"))
+
+;; SPDX-License-Identifier: GPL-3.0-or-later
```

Comment:
Change this to 'Apache License 2.0` and remove the body of the license. We have a [LICENSE](https://github.com/Orange-OpenSource/hurl/blob/master/LICENSE) file that is the unique reference of the whole project.